stable/msoms/templates/omsagent-daemonset.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
```

Comment:
Standard labels are missing: `app`, `chart`, `heritage`, `release`.

Reply:
This has been addressed